tests/testthat/output/glimpse/iris-70-na-nrow-ncol.txt

```diff
@@ -0,0 +1,7 @@
+Observations: ?
+Variables: ?
```

Comment:
Which `"tbl"` classes return `NA` as number of columns? We iterate all columns anyway in `glimpse()`, so we know the number of variables in the output.

Reply:
That's a great point. In sparklyr getting the number of columns could be very expensive (i.e. you'd have to compute and not just pull it off metadata/schema), but in those cases computing the first few rows would also be expensive. We return `NA` for `ncol()` because we didn't want to break the expectation that `ncol()` should be fast. Let me think for a bit on how we want to deal with this...